src/reason_parser.mly

```diff
@@ -788,8 +788,17 @@ let rec string_of_longident = function
 
 let built_in_explicit_arity_constructors = ["Some"; "Assert_failure"; "Match_failure"]
 
+let jsx_component module_name attrs children ~loc =
```

Comment:
I suppose we can say we have "JSX For Reason" which is slightly different than JSX.